tools/load_veldwerk_json.py

```diff
@@ -426,7 +426,7 @@ def fielddata_to_memcollections(filename, profile_plan_col=None, profile_id_fiel
             log.warning('records toegevoegd %i', i + 1)
 
     # Extract fixed points from the geojson
-    for fp_pk, point_ids in enumerate(json_dict['point_notes']):
+    for fp_pk, point_ids in enumerate(json_dict.get('point_notes', [])):
```

Comment:
Ik zag net in de testdata dat het een dictionary is. 
Volgens mij moet het dan zijn:
json_dict.get('point_notes', {}).items()  of werkt het zo ook?

Reply:
Ik heb het veranderd en wat extra tests toegevoegd om zowel een lege als een niet bestaande point_notes te testen. Ik vermoed dat het daarvoor ook gewoon werkte, ik ondervond geen problemen. Dit is wel mooier natuurlijk.